src/v/cluster/health_monitor_types.h

```diff
@@ -171,8 +171,8 @@ struct node_health_report
 struct cluster_health_report
   : serde::envelope<
       cluster_health_report,
-      serde::version<0>,
-      serde::compat_version<0>> {
+      serde::version<1>,
```

Comment:
This may need to be behind a feature manager check

Reply:
are you trying to resolve this before the next review?

Reply:
I actually don't believe this must be behind feature manager, i've since updated. The update makes compat_version 0